app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Following the ERD and your schema, this association should be:

```suggestion
  belongs_to :author, class_name: 'User'
```

Here, we are enabling the `Post.author` method instead of `Post.user` which is less descriptive for the context of a post! 🤓 🔧

Reply:
Thanks @anagudelogu for pointing this out. I will make the required change.